src/components/MessageActions/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './MessageActions';
 export * from './MessageActionsBox';
 export * from './CustomMessageActionsList';
+export * from './MessageActions.new';
```

Comment:
I was thinking, what if instad we export a bunch of components/objects with `_UNSTABLE` prefix, we do the opposite?
```ts
// in the sdk
export * as MessageActions_UNSTABLE from './MessageActions.new';

// and then use:
import { MessageActions_UNSTABLE } from 'stream-chat-react';
const { MessageActions,  defaultMessageActionSet } = MessageActions_UNSTABLE;

const MyComponent = () => {
  return <MessageActions />;
}
```

Once this change becomes "stable", we can release it without making too many renames in the SDK, and integrators can fix this by updating the imports only/

Reply:
This is way better, will do. :)